images/timer.proto

```diff
@@ -21,7 +21,8 @@ message posix_timer_entry {
 	required uint64		insec		= 8;
 	required uint64		vsec		= 9;
 	required uint64		vnsec		= 10;
-	required int32		notify_thread_id= 11;
+
+	optional int32		notify_thread_id= 11;
```

Comment:
Should we fix this in https://github.com/checkpoint-restore/criu/pull/1532/commits/875c5434c2fee8b65e118c4b91c6ef175b9f9511 instead of creating a new patch/commit?